src/main/java/edu/cornell/kfs/sys/util/EnumConfiguredMappingStrategy.java

```diff
@@ -47,7 +48,7 @@ private void initializeOrderedPojoPropertyNameMappings() {
     }
 
     @Override
-    public String[] generateHeader() {
+    public String[] generateHeader(T bean) throws CsvRequiredFieldEmptyException{
```

Comment:
Missing space between exception classname and curly brace.

Reply:
updated